.docksal/commands/composer

```diff
@@ -1,15 +0,0 @@
-#!/usr/bin/env bash
```

Comment:
Do we no longer need the fin composer?  My assumption is that it is in docksal core now.